src/Quartz.Tests.Unit/Simpl/TaskSchedulingThreadPoolTest.cs

```diff
@@ -0,0 +1,77 @@
+using NUnit.Framework;
+using Quartz.Simpl;
+
+namespace Quartz.Tests.Unit.Simpl
```

Comment:
nit: can use file-scoped namespaces for new files (no merge problems)

Reply:
Perhaps consider enabling analyzers? Do you want me to submit a PR for this?
Do you want me to change the entire code base to file-scoped namespaces?

Reply:
We cannot change the whole code base as it would make merging between 3.x and main very tedious. So such change would be there in the farther future. This is why I noted that new code can use file-scoped namespaces as doesn't cause merge problems.